app/views/meeting.details.html

```diff
@@ -1,119 +1,93 @@
+        <!-- SECONDARY BLOCK START -->
+        <div class="db-block ng-class:[{'db-block-close db-brd-all':c.bm === c.bms.PRIMARY},{'db-block-open':c.bm !== c.bms.PRIMARY},{'db-margin-05':!isMobile}]">
 
-            <!-- SECONDARY  BLOCK HEADER -->
-            <div class="btn-group db-block-hdr db-prim-bg db-brd-top db-padding-2">
+            <div ng-if="isMobile && c.header" class="list-group db-no-margin">
+                <H5 class="db-list-group-item-prim db-truncate" ng-cloak>{{c.header}}</H5>
+            </div>
 
-                <button type="button" class="btn db-btn-prim db-btn-small db-brd db-margin-2" ng-disabled="parallelMode" ng-click="c.secondaryClicked()">
+            <!-- SECONDARY BLOCK HEADER -->
+            <div class="db-prim-bg ng-class:[{'list-group db-no-margin db-padding-10 db-brd-all':isMobile},{'btn-group db-flex-wrap db-brd-top db-padding-2':!isMobile}]">
+                <button ng-if="menuClosed() && c.pm === c.pms.HIDDEN" type="button" class="btn db-btn-prim db-btn-small db-brd-all db-margin-2" ng-click="openMenu()">
+                    <span class="glyphicon glyphicon-menu-hamburger"></span>
+                </button>
+                <button ng-if="c.pm !== c.pms.HIDDEN" type="button" class="btn btn-block db-btn-prim db-btn-small db-brd-all ng-class:[{'db-margin-2':!isMobile}]" ng-disabled="parallelMode" ng-click="c.secondaryClicked()">
                     <span class="glyphicon ng-class:[{'glyphicon-minus':c.bm === c.bms.SECONDARY},{'glyphicon-plus':c.bm !== c.bms.SECONDARY}]"></span>
                 </button>
-
-                <button type="button" class="btn db-btn-sel db-btn-big db-brd db-margin-2 ng-class:[{'active':c.lbm === c.lbms.ATTACHMENTS}]"
-                    ng-disabled="!c.materialCount()" db-confirm confirm-enabled="c.hasUnsavedProposal || c.remarkIsUnsaved" confirm-config="c.unsavedConfig"
-                    ng-click="c.attachmentClicked()">
+                <button ng-if="c.pm === c.pms.HIDDEN" type="button" class="btn btn-block db-btn-sel db-btn-big db-brd-all ng-class:[{'db-margin-2':!isMobile},{'active':c.sm === c.sms.SECRET}]" ng-disabled="!c.tData.link" ng-click="c.presentationClicked(c.tData)">
+                    <span translate translate-cloak>STR_PRESENTATION</span>
+                </button>
+                <button type="button" class="btn btn-block db-btn-sel db-btn-big db-brd-all ng-class:[{'active':c.sm === c.sms.ATTACHMENTS},{'db-margin-2':!isMobile}]" ng-disabled="!c.materialCount()" db-confirm confirm-enabled="c.hasUnsavedProposal || c.remarkIsUnsaved"
+                    confirm-config="c.unsavedConfig" ng-click="c.attachmentClicked()">
                     <span translate translate-cloak>STR_ATTACHMENTS</span>
-                    <span class="badge ng-cloak ng-class:[{'db-font-clr': c.materialCount()}]">{{c.materialCount()}}</span>
+                    <span class="badge ng-cloak ng-class:[{'db-font-clr-dark': c.materialCount()}]">{{c.materialCount()}}</span>
```

Comment:
Why not ng-hide/ng-if the badge instead of styling it based on materialCount?
Also, there's `db-badge-hdr`  which offers style for badges placed into headers.
Should  a similar `db-badge-btn` be created for buttons inside badges?